depends/patches/native_qt/qt5-tablet-osx.patch

```diff
@@ -0,0 +1,22 @@
+Needs to be reviewed once OS X support is added.
```

Comment:
If we upgrade past 5.2.1 at some point, this patch can, in all likelihood, be removed. The code's in 5.4.1, for example.